instructions/git.md

```diff
@@ -0,0 +1,28 @@
+To follow along with the Git lesson, we will be using RStudio Cloud. Please 
+follow these instructions carefully.
+
+1. Navigate your browser to <https://rstudio.cloud> and click sign up or login
```

Comment:
I think it might be worth splitting the instructions between (1) creating an account; (2) logging in (just in case people already have an account).

Reply:
Yes! Done